src/sites/twitch-twilight/modules/player.jsx

```diff
@@ -112,7 +112,7 @@ export default class Player extends Module {
 			}
 		});
 
-		if ( document.pictureInPictureEnabled )
+		if ( document.pictureInPictureEnabled || !!window.safari )
```

Comment:
Rather than introducing an or statement like this everywhere, please use a helper method or property such as the `isPiPAvailable` that you added but commented out.